src/slider.jsx

```diff
@@ -559,10 +560,10 @@ const Slider = React.createClass({
           onMouseLeave={this.handleMouseLeave}
           onMouseUp={this.handleMouseUp}
         >
-          <div ref="track" style={this.prepareStyles(styles.track)}>
-            <div style={this.prepareStyles(styles.filled)}></div>
-            <div style={this.prepareStyles(remainingStyles)}></div>
-            <div style={this.prepareStyles(handleStyles)} tabIndex={0} {...handleDragProps}>
+          <div ref="track" style={prepareStyles(styles.track)}>
+            <div style={prepareStyles(styles.filled)}></div>
+            <div style={prepareStyles(remainingStyles)}></div>
+            <div style={prepareStyles(handleStyles)} tabIndex={0} {...handleDragProps}>
```

Comment:
On [L37](https://github.com/callemall/material-ui/pull/3332/files#diff-55f0973adedf81bc6eee98d811bf2729R537), we can save an allocation by taking the empty object off.

```js
 let handleDragProps; // = {}; removed
 
     if (!this.props.disabled) {
       handleDragProps = {
         onTouchStart: this._onHandleTouchStart,
         onMouseDown: this._onHandleMouseDown,
        };
      }

// Or write as
const handleDragProps = !this.props.disabled && {
  onTouchStart: this._onHandleTouchStart,
  onMouseDown: this._onHandleMouseDown,
};
```